emails/joining_reminder_email.Rmd

```diff
@@ -14,16 +28,22 @@ params:
 We're looking forward to welcoming you to the RSE Sheffield training session on **git & GitHub through GitKraken Client
  - from Zero to Hero** shortly!
 
+### SCHEDULE
 
-To join the session please click the link below:
- 
+We’ll **kick off at `r params$event_start_time_day1` on `r format(strptime(params$day1, format = "%Y-%m-%d"), "%a %b %d
+%Y")`** in **`r params$venue_day1`** and aim to have a few breaks throughout the morning, finishing
+at `r params$event_end_time_day1`. The second session will follow **at `r params$event_start_time_day2` on
+`r format(strptime(params$day2, format = "%Y-%m-%d"), "%a %b %d %Y")`**  in **`r params$venue_day1`** and follow the
```

Comment:
```suggestion
`r format(strptime(params$day2, format = "%Y-%m-%d"), "%a %b %d %Y")`**  in **`r params$venue_day2`** and follow the
```

Reply:
Thanks for spotting that @tdjames1 , one day I'll starting paying closer attention to my copy and pastes.